index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>My Project</title>
+</head>
+<body>
+ 
+    <div class="homepage">
+
+        <div class="images">
+            <img class="ellipse3" src="./Assets/Ellipse 3.png" alt="">
+            <img class="ellipse4" src="./Assets/Ellipse 4.png" alt="">
+            <img class="ellipse5" src="./Assets/Ellipse 5.png" alt="">
+            <img class="ellipse8" src="./Assets/Ellipse 8.png" alt="">
+            <img class="ellipse11" src="./Assets/Ellipse 11.png" alt="">
+            <img class="ellipse12" src="./Assets/Ellipse 12.png" alt="">
+            <img class="ellipse13" src="./Assets/Ellipse 13.png" alt="">
+            <img class="ellipse14" src="./Assets/Ellipse 14.png" alt="">
+            <img class="ellipse15" src="./Assets/Ellipse 15.png" alt="">
+            <img class="rectangle4" src="./Assets/Rectangle 4.png" alt="">
+            <img class="rectangle5" src="./Assets/Rectangle 5.png" alt="">
+            <img class="rectangle6" src="./Assets/Rectangle 6.png" alt=""> 
+            <img class="rectangle7" src="./Assets/Rectangle 7.png" alt="">
+        </div>
+
+        <nav>
+            <div class="logo">
+                <img class=""logo-img src="/Assets/Logo Placeholder.png" alt="My Logo">
+            </div>
+            <div class="menu-icon">
+                <img src="/Assets/Icon - Menu.png" alt="Click Menu">
+            </div>
+        </nav>
+
+        <div class="text-container">
+            <div class="main-text">I’m Mariane
+                Glad to see you!</div>
+            <div class="secondary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+        </div>
```

Comment:
- [ ] Kindly wrap your code in a `p` tag as this is a paragraph text 🙏 . This way will be semantically correct.